app/views/sites/api_instructions/_show_web_md.html.haml

```diff
@@ -1,485 +1,77 @@
-
-
-
-
-    It uses the same required parameters. All optional parameters are supported.
-
-
-
-
-    `#{api_scheme_and_host}/api/v2/search/i14y?affiliate=#{h(@site.name)}&access_key=#{h(@site.api_access_key)}&query={YOUR_SEARCH_TERM}`
-
-
-
-
-  ## Additional Instructions for Commercial API Keys
-
-
-  If you’ve [bought an API key from Bing](#{commercial_keys_site_api_instructions_path(@site)}), follow the additional instructions below to return their results.
-
-
-
-
-  ### Microsoft Cognitive Services - Bing Search API Web Results
-
-    The endpoint is `#{api_scheme_and_host}/api/v2/search/azure/web`.
-
-    In addition to the required and optional parameters above, api_key is required.
-    Your key must have a subscription for Cognitive Services Bing Search APIs.
-
-    `#{api_scheme_and_host}/api/v2/search/azure/web?affiliate=#{h(@site.name)}&access_key=#{h(@site.api_access_key)}&query={YOUR_SEARCH_TERM}&api_key={YOUR_API_KEY}`
-
-    Note that the response only includes results for the Web module.
-
-
-  ### Microsoft Cognitive Services -- Bing Search API Image Results
-
-    The endpoint is `#{api_scheme_and_host}/api/v2/search/azure/image`.
-
-    In addition to the required and optional parameters above, api_key is required.
-    Your key must have a subscription for Cognitive Service Bing Search APIs.
-
-    `#{api_scheme_and_host}/api/v2/search/azure/image?affiliate=#{h(@site.name)}&access_key=#{h(@site.api_access_key)}&query={YOUR_SEARCH_TERM}&api_key={YOUR_API_KEY}`
-
-    Note that the response only includes results for the Image module.
-
-  #### Image Response Fields
-
-    * Each image result has the following fields:
-
-    | Values                 | Description
-    | :--                    | :--
-    | title                  | Title of the image
-    | url                    | URL of the image
-    | media_url              | URL of the page containing the image
-    | display_url            | display version of URL of the page containint the image
-    | content_type           | Image file format
-    | file_size              | Number of bytes in the image
-    | width                  | Image pixel width
-    | height                 | Image pixel height
-    | thumbnail:url          | URL of the thumbnail
-    | thumbnail:content_type | File format of the thumbnail (always null as of 2017)
-    | thumbnail:file_size    | Number of bytes in the thumbnail (always null as of 2017)
-    | thumbnail:width        | Thumbnail pixel width
-    | thumbnail:height       | Thumbnail pixel height
-
-
-
+  For instance, if you set the query `example` to route to `https://www.search.gov`, then the expected response will be the following:
 
+  `{"route_to":"https://www.search.gov/"}`
 
+  To fully implement this, you will have to add logic that will process this response and redirect the user. A site-specific API call and javascript sample are available within the  Search Results API Instructions.
```

Comment:
Maybe make the 'Search Results API Instructions' at the end of this line a link?